examples/socketmode/events.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/slack-go/slack"
+	"github.com/slack-go/slack/slackevents"
+)
+
+// You more than likely want your "Bot User OAuth Access Token" which starts with "xoxb-"
+var api = slack.New("TOKEN")
```

Comment:
I know it's a bit confusing but the token that is supposed to be used for `apps.connections.open` is not a bot/user token (xoxb-/xoxp-). The token type for it is "App-Level Token" (xapp-). Thus, Socket Mode apps usually use two tokens, app token (xapp-) for apps.connections.open only plus a bot/user token.